docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1141 @@
+{
```

Comment:
<p>1) Would adding `doIHaveALawn` to the base class introduce any additional state?</p><p><br></p><p>2) Also, am I right in thinking that overriding via extensions doesn't support dynamic dispatch? I think it would be good to point this out.</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>